layers/+distribution/spacemacs-base/keybindings.el

```diff
@@ -50,6 +50,14 @@
 ;; let TAB jump between links in help buffers
 (evil-define-key 'motion help-mode-map (kbd "<tab>") 'forward-button)
 (evil-define-key 'motion help-mode-map (kbd "S-<tab>") 'backward-button)
+;; let n and p to navigate between pages
+(evil-define-key 'motion help-mode-map (kbd "n") 'help-go-forward)
+(evil-define-key 'motion help-mode-map (kbd "p") 'help-go-back)
```

Comment:
But now `n` is also used by evil for searching :stuck_out_tongue_winking_eye: 

Simply keep `[` and `]` for sure, and `s`, `u` and `d` if you think they are useful :relaxed: